mitiq/qiskit/conversions.py

```diff
@@ -0,0 +1,66 @@
+"""Functions to convert from Mitiq's internal circuit representation
```

Comment:
Cool! This whole module does come out rather smoothly. Seems to validate the choice of Cirq as at least a relatively easy place to start.